Dockerfile

```diff
@@ -1,11 +1,14 @@
 # Install the app dependencies in a full Node docker image
 FROM registry.access.redhat.com/ubi8/nodejs-16:latest
 
-# Copy package.json and package-lock.json
-COPY package*.json ./
+# Copy package.json, and optionally package-lock.json if it exists
+COPY package.json package-lock.json* ./
```

Comment:
The final `npm start` command in the `Dockerfile` relies on a `package.json` existing. This command means we fail early if it doesn't.

Reply:
I assume the * at the end is what gets us the does not fail if package-lock.json does not exist but it would copy over package-lock.json2 etc. as well. We might be able to tighten that up but I don't think that is a blocker for this PR.